### PR TITLE
Added Equity to the player page

### PR DESCRIPTION
**Pull request by berong91**
Portfolio page now shows the player's Equity and Cash. Formats the number so it looks better.

### CODE REVIEW

File: application/config/database.php [hunk 1]
```diff
@@ -77,8 +77,8 @@
 	'dsn'	=> '',
 	'hostname' => 'localhost',
 	'username' => 'root',
-	'password' => '',
```
Comment (brandondtran) [inline application/config/database.php]: This can't be changed

File: application/models/Players.php [hunk 2]
```diff
@@ -34,11 +34,19 @@ function headers()
 
     public function get($id)
     {
-        $this->db->select('*');
-        $this->db->where('ID=', $id);
-        return $this->db->get('players')->result_array();
+        return $this->db->query('SELECT players.Player, players.Cash, SUM(temp.Total) AS \'Equity\' '
```
Comment (brandondtran) [inline application/models/Players.php]: This should not be in the get function

Comment (berong91) [inline application/models/Players.php]: I think we need to show the player's cash and equity in his portfolio page, and the changes is required to achieve that.

**Comment by berong91** [comment]
Changes:

- Model Players: Created method getPlayerEquity that seperates from the original get method.
- Player portfolio page: Modified the view with the new method.

**Pull request merged**